### ISSUE TITLE
Can't build in Arch Linux

### ISSUE THREAD

**Issue by arlllk** [arkane-systems/genie#20]
I get this errors when I try to compile the project

> Microsoft (R) Build Engine version 15.9.20.63311 for .NET Core
> Copyright (C) Microsoft Corporation. All rights reserved.
> 
>   Restoring packages for /home/arllk/genie/genie/genie.csproj...
>   Generating MSBuild file /home/arllk/genie/genie/obj/genie.csproj.nuget.g.props.
>   Generating MSBuild file /home/arllk/genie/genie/obj/genie.csproj.nuget.g.targets.
>   Restore completed in 7.26 sec for /home/arllk/genie/genie/genie.csproj.
> Program.cs(75,50): error CS0121: The call is ambiguous between the following methods or properties: 'CommandHandler.Create<T>(Func<T, int>)' and 'CommandHandler.Create<T>(Func<T, Task<int>>)' [/home/arllk/genie/genie/genie.csproj]
> Program.cs(80,52): error CS0121: The call is ambiguous between the following methods or properties: 'CommandHandler.Create<T>(Func<T, int>)' and 'CommandHandler.Create<T>(Func<T, Task<int>>)' [/home/arllk/genie/genie/genie.csproj]
> Program.cs(87,47): error CS0121: The call is ambiguous between the following methods or properties: 'CommandHandler.Create<T>(Func<T, int>)' and 'CommandHandler.Create<T>(Func<T, Task<int>>)' [/home/arllk/genie/genie/genie.csproj]
> Program.cs(99,46): error CS0121: The call is ambiguous between the following methods or properties: 'CommandHandler.Create<T1, T2>(Func<T1, T2, int>)' and 'CommandHandler.Create<T1, T2>(Func<T1, T2, Task<int>>)' [/home/arllk/genie/genie/genie.csproj]
> cp: cannot stat 'bin/Debug/netcoreapp2.2/linux-x64/publish/genie': No such file or directory
> cp: cannot stat 'bin/Debug/netcoreapp2.2/linux-x64/publish/*.dll': No such file or directory
> cp: cannot stat 'bin/Debug/netcoreapp2.2/linux-x64/publish/genie.runtimeconfig.json': No such file or directory
> chown: cannot access '../systemd-genie/usr/bin/*': No such file or directory
> chmod: cannot access '../systemd-genie/usr/bin/genie': No such file or directory
> chmod: cannot access '../systemd-genie/usr/bin/genie': No such file or directory

With the version
> dotnet --version
> 2.2.108

**Comment by cerebrate** [comment]
Can you try updating your .NET Core SDK? It builds fine here on 2.2.401, and I want to make sure that there aren't any SDK changes between .108 and .401 causing this.

**Comment by arlllk** [comment]
I just updated to 
```
[arllk@Arley-wsl linux-x64]$ dotnet --info
.NET Core SDK (reflecting any global.json):
 Version:   2.2.401
 Commit:    729b316c13

Runtime Environment:
 OS Name:     arch
 OS Version:
 OS Platform: Linux
 RID:         linux-x64
 Base Path:   /opt/dotnet/sdk/2.2.401/

Host (useful for support):
  Version: 2.2.6
  Commit:  7dac9b1b51

.NET Core SDKs installed:
  2.2.401 [/opt/dotnet/sdk]

.NET Core runtimes installed:
  Microsoft.AspNetCore.All 2.2.6 [/opt/dotnet/shared/Microsoft.AspNetCore.All]
  Microsoft.AspNetCore.App 2.2.6 [/opt/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 2.2.6 [/opt/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download
```
And now when I try to build it I get
```
[arllk@Arley-wsl ~]$ git clone https://github.com/arkane-systems/genie.git
Cloning into 'genie'...
remote: Enumerating objects: 203, done.
remote: Counting objects: 100% (203/203), done.
remote: Compressing objects: 100% (140/140), done.
remote: Total 203 (delta 93), reused 139 (delta 51), pack-reused 0
Receiving objects: 100% (203/203), 42.09 KiB | 8.42 MiB/s, done.
Resolving deltas: 100% (93/93), done.
[arllk@Arley-wsl ~]$ cd genie
[arllk@Arley-wsl genie]$ cd genie
[arllk@Arley-wsl genie]$ ./build
Microsoft (R) Build Engine version 16.2.32702+c4012a063 for .NET Core
Copyright (C) Microsoft Corporation. All rights reserved.

  Restore completed in 586.01 ms for /home/arllk/genie/genie/genie.csproj.
  genie -> /home/arllk/genie/genie/bin/Debug/netcoreapp2.2/linux-x64/genie.dll
  genie -> /home/arllk/genie/genie/bin/Debug/netcoreapp2.2/linux-x64/publish/
cp: cannot create regular file '../systemd-genie/usr/bin/': No such file or directory
cp: target '../systemd-genie/usr/bin/' is not a directory
cp: cannot create regular file '../systemd-genie/usr/bin/': No such file or directory
chown: cannot access '../systemd-genie/usr/bin/*': No such file or directory
chmod: cannot access '../systemd-genie/usr/bin/genie': No such file or directory
chmod: cannot access '../systemd-genie/usr/bin/genie': No such file or directory
```

**Comment by cerebrate** [comment]
Ah, yeah, that’s just the build script not being able to copy the final files into the packaging directory (because git doesn’t recreate empty directories). If you make the directories manually it’ll work.

Or don’t bother - you’re on Arch, so you’re not going to want a .deb package. You can just look at the build script for the files and perms it wants and copy them there direct.

**Comment by cerebrate** [comment]
Fixed in 1.9, although the build script now calls dkpg-deb, so please ignore errors from that.

**Comment by arlllk** [comment]
It now compiles with dotnet 2.2.108?
